charts/icm-as/templates/as-deployment.yaml

```diff
@@ -156,6 +156,11 @@ spec:
               protocol: TCP
           resources:
             {{- toYaml .Values.resources | trim | nindent 12 }}
+          lifecycle:
+            preStop:
+              httpGet:
+                port: 7743
```

Comment:
I would prefer to use the "semantic" name of the port "admin" here instead of the port number.
Also all other "admin" port usages for K8s lifecycle should not use "http" as name.

Reply:
It's changed to use the reference to the servlet port now. In terms of renaming them to "admin", it's rather part of the user story @skiesewetter-intershop is currently working on, therefore we talked and agreed that he'll cover that mentioned renaming in his PR.